universal_robot/082dcf1/082dcf1.bug

```diff
@@ -80,4 +80,4 @@ fault-codes:
   - BDO:DEPENDENCY
 failure-codes:
   - SOFTWARE:CRASHING
-  - SYSTEM:LOSS-OF-CONTROL
+  - SYSTEM:UNRESPONSIVE
```

Comment:
Loss of functionality seems more appropriate. @ipa-hsd @ChrisTimperley

Reply:
Following the reasoning above, let's go with Unresponsive and Loss of Functionality.